stories/tooltip.stories.js

```diff
@@ -0,0 +1,259 @@
+
+.${component}__Container {
+  @include mercury.${component}__Container;
+
+  &--Left {
+    @include mercury.${component}__Container--Left;
+  }
+}
+
+.${component} {
+  @include mercury.${component};
+
+  ${comment.modifier}
+
+  &--Left {
+    @include mercury.${component}--Left;
+  }
+}`,
+      }}
+    />
+  </Story>
+);
+
+export const right = () => (
+  <Story>
+    <Demo>
+      <div
+        className="Manifold__Tooltip__Container Manifold__Tooltip__Container--Right"
+        aria-describedby="tooltip-1"
+      >
+        <button type="button" className="Manifold__Button">
+          Hover me
+        </button>
+        <div role="tooltip" id="tooltip-1" className="Manifold__Tooltip Manifold__Tooltip--Right">
+          Tooltip describing the trigger element.
+        </div>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Right</h1>
+      <p>Display the tooltip to the right of the trigger</p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- ${title} Container -->
+<div class="Manifold__Tooltip__Container" aria-describedby="tooltip-1">
+  <!-- ${title} -->
+  <div role="tooltip" id="tooltip-1" class="Manifold__Tooltip Manifold__Tooltip--Right">
+    Tooltip describing the trigger element.
+  </div>
+</div>`,
+        scss: `${comment.block(title)}
+
+.${component}__Container {
+  @include mercury.${component}__Container;
+
+  &--Right {
+    @include mercury.${component}__Container--Right;
+  }
+}
+
+.${component} {
+  @include mercury.${component};
+
+  ${comment.modifier}
+
+  &--Right {
+    @include mercury.${component}--Right;
+  }
+}`,
+      }}
+    />
+  </Story>
+);
+
+export const large = () => (
+  <Story>
+    <Demo>
+      <div class="Manifold__Tooltip__Container" aria-describedby="tooltip-1">
+        <button type="button" className="Manifold__Button">
+          Hover me
+        </button>
+        <div role="tooltip" id="tooltip-1" class="Manifold__Tooltip Manifold__Tooltip--Large">
+          Tooltip describing the trigger element that will expand past one line neccessitating a
+          line break. Use sparingly! Tooltips should always be brief if possible. They are just
+          "tips" after all. 👍
+        </div>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Large</h1>
+      <p>Display a tooltip with a large amount of text that spans more than one line.</p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- ${title} Container -->
+<div class="Manifold__Tooltip__Container" aria-describedby="tooltip-1">
+  <!-- ${title} -->
+  <div role="tooltip" id="tooltip-1" class="Manifold__Tooltip Manifold__Tooltip--Right">
+    Tooltip describing the trigger element.
+  </div>
+</div>`,
+        scss: `${comment.block(title)}
+
+.${component}__Container {
+  @include mercury.${component}__Container;
+}
+
+.${component} {
+  @include mercury.${component};
+}
+
+  ${comment.modifier}
+
+  &--Large {
+    @include mercury.${component}--Large;
+  }`,
```

Comment:
This should be inside the `}` above (see the preview link)

Reply:
![Screen Shot 2020-04-15 at 13 03 26](https://user-images.githubusercontent.com/1369770/79377808-82960080-7f19-11ea-8795-29b66ef01d02.png)